custom-css.json

```diff
@@ -1,9 +1,5 @@
 {
   "properties": {
-    "-apple-color-filter": {},
```

Comment:
This isn't internal, but should be disabled by default: https://bugs.webkit.org/show_bug.cgi?id=217802